review_practice/react-app/src/components/PersonnelController/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+
+import * as Style from './style';
+
+/* 
+personnelItems: [state, setState, rendering되는 이름, payload에 담길 이름]
+personnelControlled: 버튼클릭 시 호출되는 함수
+*/
+export default ({ personnelItems, requestToServer, personnelControlled, deleteButtonHandle }) => {
+    const saveButtonHandler = () => () => {
+        const personnel = personnelItems.reduce((payload, item) => {
+            const [stateValue, stateName] = [item[0], item[3]];
+            payload[stateName] = stateValue;
+            return payload;
+        }, {});
+        requestToServer(personnel);
+    };
+
+    const decreaseButtonHandler = (state, setState) => () => {
+        if (state === 0) return;
+        setState(state - 1);
+    };
+
+    const increaseButtonHandler = (state, setState) => () => {
+        setState(state + 1);
+    };
+
+    useEffect(() => {
+        const states = personnelItems.map(item => item[0]);
+        personnelControlled(states);
+    });
+
+    return (
+        <Style.PersonnelController>
+            {personnelItems.map((state, index) => (
```

Comment:
state[0], state[1], state[2] 가 의미하는 바를 명확하게 드러낼 수 있다면 좋겠습니다.
```suggestion
            {personnelItems.map(([state, setState, rendering되는 이름, payload에 담길 이름], index) => (
```